addons/packages/multus-cni/3.7.1/test/go.sum

```diff
@@ -1,8 +1,10 @@
 github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
Noticed the same issue as mentioned in https://github.com/vmware-tanzu/community-edition/pull/2444/files#r743377662